app/scripts/modules/core/src/managed/EnvironmentRow.tsx

```diff
@@ -30,8 +32,8 @@ export function EnvironmentRow({ name, isProd = false, children }: IEnvironmentR
           <span className={envLabelClasses}>{name}</span>
         </span>
         <div className="expand" onClick={() => setIsCollapsed(!isCollapsed)}>
-          {isCollapsed && <i className="ico icon-expand" />}
-          {!isCollapsed && <i className="ico icon-collapse" />}
+          {isCollapsed && <Icon name="accordionExpand" size="medium" />}
+          {!isCollapsed && <Icon name="accordionCollapse" size="medium" />}
```

Comment:
@gcomstock you'll notice in the "After" screenshot on this PR that these expand/collapse icons changed from 'production' red to the neutral grey icon color — I did this because I'd prefer to stick with the three color options we chose for `<Icon/>` last week until we can come up with a scalable solution to additional color permutations. Alternatively we do have the option to decide that trying to standardize on 3 options was too restrictive for our ideal design system and we should just let people pick any available color 😆

Reply:
Sounds good!